core/vm/opcodes.go

```diff
@@ -63,6 +63,9 @@ const (
 	XOR
 	NOT
 	BYTE
+	SHL
+	SHR
+	SAR
```

Comment:
Ah, missed this previously. You'll need to add these opcodes to the `opcode` -> `string` map in the same file a few hundred lines below this into `opCodeToString`.